src/Symfony/Component/Form/Guess/ValueGuess.php

```diff
@@ -18,19 +18,14 @@
  */
 class ValueGuess extends Guess
 {
-    /**
-     * The guessed value.
-     *
-     * @var array
-     */
     private $value;
 
     /**
      * Constructor.
      *
-     * @param string $value      The guessed value
-     * @param int    $confidence The confidence that the guessed class name
-     *                           is correct
+     * @param string|int|bool $value      The guessed value
```

Comment:
`null` value spotted as well :) https://github.com/symfony/symfony/blob/v3.2.8/src/Symfony/Bridge/Doctrine/Form/DoctrineOrmTypeGuesser.php#L135

Cant we allow `scalar|null`?